tests/stubs.py

```diff
@@ -1,10 +1,13 @@
+
 from enum import IntEnum
+from io import BytesIO
 from multiprocessing import Process
+from pathlib import Path
 
 from flask import Flask, request
+from PIL import Image
 from requests import get
 from requests.exceptions import ConnectionError
-
```

Comment:
The test did not deserve that extra blank line?

Reply:
Ran yapf, line is back.